spec/client_configuration_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'spec_helper'
 
+def os_info
```

Comment:
Instead of recreating the info I suggest using a regex to see if the output matches the spec.